inference-engine/src/transformations/src/transformations/common_optimizations/optimize_strided_slice.cpp

```diff
@@ -225,6 +225,11 @@ bool ngraph::pass::GroupedStridedSliceOptimizer::run_on_function(std::shared_ptr
         NodeVector ops_to_replace;
         for (auto & record : output_to_size) {
             if (record.first != fake_output) {
+                const auto out_name = record.first.get_node_shared_ptr()->get_friendly_name();
+                NGRAPH_SUPPRESS_DEPRECATED_START
+                variadic_split->get_output_tensor(i).set_name(out_name);
+                NGRAPH_SUPPRESS_DEPRECATED_END
```

Comment:
Discussed offline with @ilyachur  that in this case when at least one StridedSlice has Result operation as a consumer we can not preserve it's name. And in other transformations when do some checks to preserve names. 
In this particular transformation this check is missing, so instead of using deprecated API I suggest to update this pass to avoid replacement if one of SS has Result op.

Reply:
Thank You, I decided to follow your suggestion - please take a look.